src/main/resources/content/k8s/do-not-share-host-network-with-containers.json

```diff
@@ -1 +1,8 @@
-{"severity":"high","rule":"$.spec.template.spec.hostNetwork is true","id":"99544e17-fc8f-4c77-963e-083ab80c53b0 ","enabled":true,"resourceType":"k8s","policy":"do not share host network with containers"}
\ No newline at end of file
+{
+  "severity": "high",
+  "rule": "$.spec.template.spec.hostNetwork is true",
+  "id": "99544e17-fc8f-4c77-963e-083ab80c53b0 ",
+  "enabled": true,
+  "resourceType": "k8s",
+  "policy": "do not share host network with containers"
```

Comment:
description field missing here as well.
Also, the policy name is not sentence case.